package/network/utils/wireguard-tools/Makefile

```diff
@@ -23,6 +23,8 @@ PKG_LICENSE_FILES:=COPYING
 
 PKG_BUILD_PARALLEL:=1
 
+PKG_CONFIG_DEPENDS:=CONFIG_LINUX_5_4
```

Comment:
it's not necessary to set a dependency here.

Reply:
What about in package/network/services/wireguard/Makefile, also not necessary?

Reply:
Yes